malware/system/ksp-audit-xorddos-process.yaml

```diff
@@ -0,0 +1,39 @@
+# KubeArmor is an open source software that enables you to protect your cloud workload at run-time.
+# To learn more about KubeArmor visit:
+# https://www.accuknox.com/kubearmor/
+
+apiVersion: security.kubearmor.com/v1
+kind: KubeArmorPolicy
+metadata:
+  name: ksp-audit-xorddos-process
+  namespace: default
+spec:
+  tags: ["MALWARE", "XORDDOS"]
+  message: "Alert! good binary detected possible xorddos malware"
+  selector:
+    matchLabels:
+      container: xorddos
+  process:
```

Comment:
@yasin-cs-ko-ak  it would be better to audit the processes used by the malware rather than auditing entire folders, it'll in turn minimize the noise.